### PR TITLE
feat(client): add project_languages methode

### PR DESCRIPTION
**Pull request by fe80**
Hello,

This feature add the support of languages endpoint of a specific project.

https://docs.gitlab.com/ee/api/projects.html#languages

### CODE REVIEW

**Comment by fe80** [comment]
The lint fail because the module have too many line, I need to split into a new module or would you like I ignore this rubocop check for this module ?

**Comment by NARKOZ** [comment]
> The lint fail because the module have too many line, I need to split into a new module or would you like I ignore this rubocop check for this module ?

Please ignore it.

File: spec/gitlab/client/projects_spec.rb [hunk 1]
```diff
@@ -925,4 +925,15 @@
       expect(@custom_attributes.first.username).to eq 'gitlab+deploy-token-93'
     end
   end
+
+  describe '.project_languages' do
+    before do
+      stub_get('/projects/2/languages', 'project_languages')
```
Comment (NARKOZ) [inline spec/gitlab/client/projects_spec.rb:931]: There is no fixture with this name, so the test won't pass.

**Review by NARKOZ — CHANGES_REQUESTED** [review]
Thank you for the PR. Please address the comments.

**Comment by Mohamadddddddddd** [comment]
Hi

File: spec/gitlab/client/projects_spec.rb (hunk 1 above)
Comment (fe80) [reply spec/gitlab/client/projects_spec.rb:931]: Fixed

File: spec/gitlab/client/projects_spec.rb [hunk 2]
```diff
@@ -925,4 +925,15 @@
...
+      @custom_attributes = Gitlab.project_languages(2)
+    end
+
+    it 'Return a GitlabHash with lanagues informations' do
```
Comment (NARKOZ) [inline spec/gitlab/client/projects_spec.rb:935]:
```suggestion
    it 'returns a GitlabHash with language data' do
```

File: spec/gitlab/client/projects_spec.rb [hunk 3]
```diff
@@ -925,4 +925,15 @@
       expect(@custom_attributes.first.username).to eq 'gitlab+deploy-token-93'
     end
   end
+
+  describe '.project_languages' do
+    before do
+      stub_get('/projects/2/languages', 'project_languages')
+      @custom_attributes = Gitlab.project_languages(2)
```
Comment (NARKOZ) [inline spec/gitlab/client/projects_spec.rb:932]:
```suggestion
      @project_languages = Gitlab.project_languages(2)
```

File: spec/gitlab/client/projects_spec.rb [hunk 4]
```diff
@@ -925,4 +925,15 @@
...
+    end
+
+    it 'Return a GitlabHash with lanagues informations' do
+      expect(@custom_attributes).to be_a Gitlab::ObjectifiedHash
```
Comment (NARKOZ) [inline spec/gitlab/client/projects_spec.rb:936]:
```suggestion
      expect(@project_languages).to be_a Gitlab::ObjectifiedHash
      expect(@project_languages.to_hash.keys).to contain_exactly('Ruby', 'Shell')
```

**Review by NARKOZ — APPROVED** [review]

**Comment by NARKOZ** [comment]
Thank you ❤️

**Pull request merged**